src/cli/gentest/cli.py

```diff
@@ -9,16 +9,12 @@
 from typing import TextIO
 
 import click
-import jinja2
 
 from ethereum_test_base_types import Hash
 
-from .request_manager import RPCRequest
+from .source_code_generator import get_test_source
 from .test_providers import BlockchainTestProvider
 
-template_loader = jinja2.PackageLoader("cli.gentest")
-template_env = jinja2.Environment(loader=template_loader, keep_trailing_newline=True)
-
 
 @click.command()
 @click.argument("transaction_hash")
```

Comment:
Would be nice to extend the interface to allow specification of other chains (especially testnets - we could test with sepolia, for example):
```suggestion
@click.argument("transaction_hash")
@click.argument("chainid)
```

Can be added in a separate PR! :slightly_smiling_face:

Reply:
I think that would be great feature. I have created #908. Could you add it to #866.

Reply:
Done :slightly_smiling_face: